### ISSUE TITLE
49.3 : ATB, Fed Suns OPFOR generated with clan mechs in 3025

### ISSUE THREAD

**Issue by Question2** [MegaMek/mekhq#2901]
Fed Suns OPFOR was generated with clan mechs in 3025, specifically the Lupus Prime. This should not be possible. ATB settings were set to use "RAT Generator".

![image](https://user-images.githubusercontent.com/1191378/136676994-712419bd-ea32-4d0e-9f80-aff331184418.png)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/7316600/mekhqlog.txt)
[b.zip](https://github.com/MegaMek/mekhq/files/7316603/b.zip)

[mechfiles.zip](https://github.com/MegaMek/mekhq/files/7316602/mechfiles.zip)

**Comment by Duuvian** [comment]
I recently saw the same or similar issue happen recently. However this was with civilians units in a Special Mission, where they brought IS support vehicles that debut 60 or so years in the future (the MekHQ game date was 3001)

![later debut](https://user-images.githubusercontent.com/84265834/141656640-a7114c29-32b8-4ce3-8721-6026bc4169b3.png)

The debut date of the Jifty support vehicle is 3069 on Sarna.
I think the buster mech was also early, possibly others. I was looking at the Jiftys and kept one for salvage to fix up until I realized it was from the future, so I didn't think to keep any logs until after I advanced a day. Here is my most recent campaign save in case it's useful even the day after the scenario was resolved.

[Blackburn and Pryor Salvage30010505.cpnx.zip](https://github.com/MegaMek/mekhq/files/7532631/Blackburn.and.Pryor.Salvage30010505.cpnx.zip)

[customs.zip](https://github.com/MegaMek/mekhq/files/7532634/customs.zip)

I also attached the MekHQ log from the day after the scenario in  case that's useful but I don't think it is.
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/7532645/mekhqlog.txt)
